CHANGELOG.md

```diff
@@ -112,6 +112,7 @@
 - PR #6377 Update ci/local/README.md
 - PR #6383 Removed `move.pxd`, use standard library `move`
 - PR #6400 Removed unused variables
+- PR #6409 Allow CuPy 8.x
```

Comment:
I think this changelog entry is from another PR - the cupy version is being updated in this PR but that seems like a small part, and the PR # is different

Reply:
Yep, got flushed out with a `0.17` merge.